src/core/mac/data_poll_sender.hpp

```diff
@@ -285,11 +275,21 @@ class DataPollSender : public InstanceLocator, private NonCopyable
         kRecalculatePollPeriod,
     };
 
+    // Poll period under different conditions (in milliseconds).
+    static constexpr uint32_t kAttachDataPollPeriod =
+        OPENTHREAD_CONFIG_MAC_ATTACH_DATA_POLL_PERIOD; ///< Poll period during attach.
+    static constexpr uint32_t kRetxPollPeriod =
+        OPENTHREAD_CONFIG_MAC_RETX_POLL_PERIOD;      ///< Poll retx period due to tx failure.
+    static constexpr uint32_t kFastPollPeriod = 188; ///< Period used for fast polls.
+    static constexpr uint32_t kMinPollPeriod =
+        OPENTHREAD_CONFIG_MAC_MINIMUM_POLL_PERIOD;                  ///< Minimum allowed poll period.
+    static constexpr uint32_t kMaxExternalPeriod = ((1 << 26) - 1); ///< Maximum allowed user-specified period.
+                                                                    ///< i.e. (0x3FFFFF)ms, about 18.64 hours.
```

Comment:
How about removing/shortening the comments to make it align and look better. 

Here is my suggestion (which is removing most of the comments since they are repeating what can be inferred from the constant variable name itself).
```suggestion
    static constexpr uint32_t kAttachDataPollPeriod = OPENTHREAD_CONFIG_MAC_ATTACH_DATA_POLL_PERIOD;
    static constexpr uint32_t kRetxPollPeriod       = OPENTHREAD_CONFIG_MAC_RETX_POLL_PERIOD;
    static constexpr uint32_t kFastPollPeriod       = 188;
    static constexpr uint32_t kMinPollPeriod        = OPENTHREAD_CONFIG_MAC_MINIMUM_POLL_PERIOD;
    static constexpr uint32_t kMaxExternalPeriod    = ((1 << 26) - 1);  // ~18.6 hours.
```